src/cubical/CofThy.ml

```diff
@@ -98,18 +198,20 @@ struct
   let test_eqs (thy : t') eqs =
     List.for_all (test_eq thy) eqs
 
-  let test_var (thy : t') v =
-    VarSet.mem v thy.true_vars
+  let test_var (thy : t') v (b : bool) =
+    Assignment.test v b thy.known_vars
 
   let test_vars (thy : t') vs =
-    VarSet.subset vs thy.true_vars
+    Assignment.subset vs thy.known_vars
```

Comment:
What exactly does `test_vars` do? Is it checking that some little assignment is compatible with the big one or something?

Reply:
Yes---(partial) functions as sets of input-output pairs.